src/app/admin/components/reports/reports.directive.spec.js

```diff
@@ -532,24 +504,6 @@
                         expect(vm.searchedCertifiedProducts[0]).toEqual(expectedActivity);
                     });
 
-                    it('should handle criteria addition', function () {
```

Comment:
These two tests should be removed, as the system still needs to get the right output for criteria addition/removal, even if we're not treating them as questionable any more